e2e/docker-image/docker-image.test.ts

```diff
@@ -0,0 +1,32 @@
+import { createTenv } from '@e2e/tenv';
+
+const { container } = createTenv(__dirname);
+
+beforeAll(async () => {
+  const isCI = !!process.env.CI;
+  const image = isCI ? 'ghcr.io/ardatan/graphql-mesh:test' : 'graphql-mesh-gateway';
```

Comment:
I think it should be the same name for dev and ci?

Reply:
sure, fixed it